classes/db/database.class.php

```diff
@@ -849,6 +850,8 @@ public function where($table, $whereArray = null) {
 							unset($or);
 						}
 					} else {
+						// Remove column label so that it can correctly check against table columns
+						$key = (is_string($key) && $label ? preg_replace("/^$label\./", '', $key, 1) : $key);
```

Comment:
See https://github.com/briansandall/v6/commit/cddc4b4cab0d5be8f087b6247fb3ac026361cd03

Again, I'm not entirely certain under what circumstances `$whereArray` would contain arrays, so my assumptions about which element would have the label may be off.

Also, line 842 isn't handled yet; not sure if it needs to be:
```
// $key seems like it could have a $label
$or[] = "`$key` $modifier IN (".implode(',', $value).')';
```